src/test/java/core/basesyntax/service/handler/RemoveFruitHandlerImplTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.service.handler;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.fruit.dto.FruitDto;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class RemoveFruitHandlerImplTest {
+    private final FruitHandler takeFromStorage = new RemoveFruitHandlerImpl();
+    private final Map<String, Integer> actual = Storage.getFruitStorage();
+
+    @BeforeClass
+    public static void addToStorage() {
+        Storage.getFruitStorage().put("mango", 52);
+        Storage.getFruitStorage().put("cactus", 20);
+    }
+
+    @Test
+    public void handleGoods_uniqueKeys_ok() {
+        addToStorage();
```

Comment:
```suggestion
```
if you want this method to be called every time - make it `Before`, not `BeforeClass`